tests/pdarray_creation_test.py

```diff
@@ -419,6 +419,10 @@ def test_full(self):
         string_len_full = ak.full("5", 5)
         self.assertEqual(5, len(string_len_full))
 
+        strings_full = ak.full(5, "test")
+        self.assertIsInstance(strings_full, ak.Strings)
+        self.assertEqual(5, len(strings_full))
+
```

Comment:
What would happen here with a size of 0. May want to add a test case that shows a nonscalar raises and error.

Reply:
it seems to handle this correctly and returns an empty Strings
```python
>>> ak.full(0, 'str')
array([])
```